deploy/docker-compose.yml

```diff
@@ -75,8 +75,15 @@ services:
   d3-btc-node0:
     image: kylemanna/bitcoind:latest
     container_name: d3-btc-node0
-    volumes:
-      - ./bitcoin/bitcoin.conf:/bitcoin/.bitcoin/bitcoin.conf
+    entrypoint:
+    - bitcoind
+    - --rpcuser=test
```

Comment:
I suggest to make `rpcuser`, `rpcpassword`, `rpcallowip`, `rpcport` variables user-configurable via environment vars